libc/src/errno/libc_errno.h

```diff
@@ -14,43 +14,17 @@
 
 #include <errno.h>
 
-// If we are targeting the GPU we currently don't support 'errno'. We simply
-// consume it.
-#ifdef LIBC_TARGET_ARCH_IS_GPU
+// This header is to be consumed by internal implementations, in which all of
+// them should refer to `libc_errno` instead of using `errno` directly from
+// <errno.h> header.
```

Comment:
Can you add a sentence about hermetic and integration tests?
AFAIU they should:
 - Depend on `libc.src.errno.errno`
 - Not `#include "src/errno/libc_errno.h"`
 - But `#include <errno.h>`
 - Use regular `errno` in the code.

Reply:
Done.